pastis/fields_building_numerical_all_laurent.py

```diff
@@ -0,0 +1,445 @@
+"""
+This module contains functions that calculates the field sensitivities for a given coronagraph design
+"""
+
+import os
+import time
+import functools
+from shutil import copy
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.io import fits
+import astropy.units as u
+import hcipy as hc
+
+from config import CONFIG_INI
+import util_pastis as util
+from e2e_simulators.luvoir_imaging_onephot import LuvoirAPLC
+
+
+def num_fields_luvoir(design, savepsfs=True, saveopds=True):
```

Comment:
@ananyasahoo1904 does this function work, have you tested it? It is based on an outdated version of building the matrices and some of the imports up top are broken, so we should test this and fix as appropriate before merging.
Also, how does this differ from the notebook with the same name?